packages/core/src/LernaTool.ts

```diff
@@ -0,0 +1,49 @@
+import path from 'path';
+import fs from 'fs-extra';
+
+import { PackageJSON } from "@changesets/types";
+import { Tool, ToolType, Package, Packages, MonorepoRoot } from './Tool';
+import { expandPackageGlobs } from "./expandPackageGlobs";
+
+export const LernaTool: Tool = {
```

Comment:
Open to playing with satisfies! I wouldn't want to lose the "hard typing" of the object to Tool though, as a tool author, to get that instant red feedback in VSCode if I get out of alignment with the API as I'm writing my method.

Reply:
I think it would check all the boxes for us here - you can play around with it [here](https://www.typescriptlang.org/play?#code/JYOwLgpgTgZghgYwgAgCoHt0BtkG8BQyRyUEcAJuiFgJ7Jg0AOEAXMgM5hSgDmA3IWLB2AWSrpSjdACVMYABTlgpBGAk02nbiB4BKNgAUo6ALbCIAHgBGmLGRAA+AcWTCxICRCmz0YAMo0IAiKyhCq6ppcvPrINtj2AgC++PgIVJzIADLQIHAY2MgAvHiCDMxsAOR2ULkVyHDsyGkgnAA0gg2BCK6i4pIycvK6JS6kYACuNfRQ4xCCie1Ebn1eA74BQUMjxGOTINOz8-iJHHBgwjDAEI35WCkA9PfIjFhwdHDG4yDkyADuwGAABb1cZqNImF4QSCNODfDgTGAwfDZGp5WwAOhSzQyBhAjBMtyK2zKrGQFUYeJMdQaTXSYEW9XYXR67k83kGwwIoyhewOcyIC2Op3O7Eu1zQtmQj2Q0GMUEa-SgkB+IGASFoQA). It would require us to bump some dev deps and I created a PR for it [here](https://github.com/Thinkmill/manypkg/pull/152)